src/app/modules/main/module.nim

```diff
@@ -353,7 +353,9 @@ method load*[T](
   self.walletSectionModule.load()
 
   # Set active section on app start
-  self.setActiveSection(activeSection)
+  # If section is profile then open chat by default
+  if activeSection.sectionType != SectionType.ProfileSettings:
```

Comment:
@saledjenic I added else block.

Reply:
@endulab what about [this](https://github.com/status-im/status-desktop/pull/5903#discussion_r883549770)?

Reply:
@saledjenic I am not convinced about adding this change because:
1. It will provide discrepancy between the current UI state and saved state, which can be potentially dangerous.
2. We are missing information about the real state when the application was closed. Maybe someone would like to read it somewhere else and do some actions.
3. It requires additional logic, because we can't just omit saving state when Profile is active - in case of: 1. go to wallet 2. go to profile 3. close the app, the saved state will be wallet, which is wrong. In this case in case we need to save Chat state. And this will lead to 1.
4. I can't find benefits of this additional change. Please write some arguments, why this is needed or maybe the current solution can be dangerous in some cases.

Reply:
> It requires additional logic, because we can't just omit saving state when Profile is active - in case of: 1. go to wallet 2. go to profile 3. close the app, the saved state will be wallet, which is wrong. In this case in case we need to save Chat state. And this will lead to 1.

@endulab actually that was the idea I proposed, to skip saving "profile settings", but store to a local file either `""` (empty string), that means if there is no stored section you should display the default one which is "Chat". And I don't see how that is going to produce discrepancy since reading stored section from the ini file will be done only on the app start (when we load the app state) and never again during that app run.

Reply:
@saledjenic if the saved section is used only in this one place, then it's ok. I updated the pr.

Reply:
@saledjenic if the saved section is used only in this one place, then it's ok. I updated the pr.